services/skipchain/api.go

```diff
@@ -0,0 +1,130 @@
+package skipchain
+
+import (
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/sda"
+)
+
+// Client is a structure to communicate with the Skipchain
+// service from the outside
+type Client struct {
+	*sda.Client
+}
+
+// NewClient instantiates a new client with name 'n'
+func NewClient() *Client {
+	return &Client{Client: sda.NewClient("Skipchain")}
+}
+
+// CreateRootInter creates two Skipchains: a root SkipChain with
+// maximumHeight of maxHRoot and an intermediate SkipChain with
+// maximumHeight of maxHInter. It connects both chains for later
+// reference.
+func (c *Client) CreateRootInter(elRoot, elInter *sda.EntityList, maxHRoot, maxHInter int, ver VerifierID) (root, inter *SkipBlock, err error) {
```

Comment:
Finally the `Propose` == `Create/Add` did only hold up in the service but was badly reflected in the API. So now we DO have
`CreateRoster`
`ProposeRoster`
`CreateData`
`ProposeData`

mostly because in the case of `Create` we need extra arguments. So instead of cramming it all in 2 usage patterns of one function, I created two functions.